plugin.xml

```diff
@@ -7,6 +7,6 @@
     <author>Ignite Realtime</author>
     <version>${project.version}</version>
     <date>04/26/2019</date>
-    <minServerVersion>4.0.0</minServerVersion>
+    <minServerVersion>4.5.0</minServerVersion>
```

Comment:
This change is probably not necessary now the API changes are compatible (and apologies for making you change it!)

Reply:
I agree with the actual changes it is compatible. 

But if we change the use of getMaxCacheSize on https://github.com/igniterealtime/openfire-hazelcast-plugin/blob/master/src/java/org/jivesoftware/openfire/plugin/util/cache/ClusteredCacheFactory.java#L478 to getLongMaxCacheSize it throws errors:

```
Exception in thread "Cache Stats" java.lang.NoSuchMethodError: org.jivesoftware.util.cache.Cache.getLongCacheSize()J
	at org.jivesoftware.openfire.plugin.util.cache.ClusteredCacheFactory.updateCacheStats(ClusteredCacheFactory.java:478)
	at org.jivesoftware.util.cache.CacheFactory$2.run(CacheFactory.java:822)
```

Reply:
How about something like 

```
if(cache instanceof ClusteredCache) {
  info[0] = ((ClusteredCache)cache).getLongCacheSize();
} else {
  // Cache.getLongCacheSize was introduced in OF 4.5, so use the old API for compatability for now
  info[0] = cache.getCacheSize();
}
```

Reply:
Works great! Thanks for the advice!